engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/DevProcessConfigCreator.scala

```diff
@@ -225,7 +225,8 @@ class DevProcessConfigCreator extends ProcessConfigCreator {
     )
   }
 
-  override def exceptionHandlerFactory(config: Config) = ParamExceptionHandler
```

Comment:
I would leave it in dev. Sometimes we want to test  e.g. parameter validation or sth?